comparisontool/management/commands/load_school.py

```diff
@@ -11,33 +13,39 @@ class Command(BaseCommand):
     args = '<path>'
```

Comment:
I think the inclusion of `args` to the `Command` class [is deprecated after 1.8](https://docs.djangoproject.com/en/1.9/howto/custom-management-commands/#django.core.management.BaseCommand.args), so is no longer necessary.